### PR TITLE
Add GitHub actions as CI

### PR DESCRIPTION
**Pull request by r0mflip**

### CODE REVIEW

**Comment by r0mflip** [comment]
Zeit installed on repo. I think Netlify can be removed. GH Actions are running way faster than Travis.

**Comment by r0mflip** [comment]
**Warn**: Removing Travis /cc @cbitosc/sc-2019 @cbitosc/sc-2018

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Node CI
```
Comment (notlmn) [inline .github/workflows/ci.yaml]:
```suggestion
name: CI
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Install packages
+      run: |
+        npm install -g @ramlmn/serv@0.8.0
+        npm install -g lighthouse@5.5.0
+    - name: Setup chrome
```
Comment (notlmn) [inline .github/workflows/ci.yaml]:
```suggestion
    - name: Setup Chrome for Lighthouse
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      run: |
+        export CHROME_PATH="$(pwd)/chrome-linux/chrome"
+        npm explore -g lighthouse -- ./lighthouse-core/scripts/download-chrome.sh
+        mv $(npm root -g)/lighthouse/chrome-linux ./
```
Comment (notlmn) [inline .github/workflows/ci.yaml]:
```suggestion
        mv "$(npm root -g)/lighthouse/chrome-linux" ./
```

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (r0mflip) [reply .github/workflows/ci.yaml]: It's node based, so...

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (r0mflip) [reply .github/workflows/ci.yaml]: Not necessarily...

Comment (notlmn) [reply .github/workflows/ci.yaml]: In this case, yes!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (notlmn) [reply .github/workflows/ci.yaml]: We also run shell scripts in here (https://github.com/cbitosc/cbitosc.github.io/pull/26/files#diff-60fe268f71ae7f8409aff40d0a3e51d8R25).

**Comment by r0mflip** [comment]
I think this PR is at its peak now.

**Pull request merged**